apps/ligretto-frontend/src/components/screens/LoaderScreen/LoaderScreen.tsx

```diff
@@ -12,7 +12,16 @@ export const LoaderScreen: React.FC = () => {
 
   return (
     <MainLayout>
-      <Box display="flex" flexDirection="column" justify-items="center" align-items="center" height="100%">
+      <Box
+        style={{
```

Comment:
Unfortunately we don't use style property (and sx) for styling.
As I know Box accept overflow property as usual prop, so we can try that: 
```suggestion
        overflow="hidden"
```

Reply:
@NewtonFernandis Could you fix it?